app/controllers/manager/users_controller.rb

```diff
@@ -24,6 +29,7 @@ def update
 
     def destroy
       @user.destroy!
```

Comment:
I think there is an exception I should have caught here

Reply:
no user's dependence is nullify so no need to catch something here